src/main/java/spoon/reflect/path/impl/CtNamedPathElement.java

```diff
@@ -62,12 +62,9 @@ public CtNamedPathElement(String pattern, boolean canBeRegexp) {
 	}
 
 	private boolean canBeRegExpPattern(String str) {
-		if (str.indexOf("()") >= 0) {
-			//this is acceptable by RegExp, but is in conflict with method signature
-			//and it makes no sense for regexp
-			return false;
-		}
-		return true;
+		//this is acceptable by RegExp, but is in conflict with method signature
```

Comment:
here the meaning of the comment is less clear. The previous version is better.

Reply:
How about this one?
```
	private boolean canBeRegExpPattern(String str) {
		//this is acceptable by RegExp, but is in conflict with method signature
		//and it makes no sense for regexp
		return str.indexOf("()") < 0;
	}
```